Src/BootCamp.Chapter/Items/Weapon.cs

```diff
@@ -1,16 +1,16 @@
 ﻿namespace BootCamp.Chapter.Items
 {
-    public class Weapon
+    public class Weapon : Item
     {
-        private string _name;
-        private decimal _price;
-        private float _weight;
+        private float _damage;
+        public float GetDamage()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider making method 'GetDamage' a property.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5138976)